proto/workload.proto

```diff
@@ -74,6 +74,36 @@ message Service {
   // Waypoint is the waypoint proxy for this service. When set, all incoming requests must go
   // through the waypoint.
   GatewayAddress waypoint = 7;
+
+  // Load balancing policy for selecting endpoints.
+  // Note: this applies only to connecting directly to the workload; when waypoints are used, the waypoint's load_balancing
+  // configuration is used.
+  LoadBalancing load_balancing = 8;
+}
+
+message LoadBalancing {
+  enum Targets {
+    UNSPECIFIED = 0;
+    REGION = 1;
+    ZONE = 2;
+    SUBZONE = 3;
+    NODE = 4;
+    CLUSTER = 5;
+    NETWORK = 6;
+  }
+  enum Mode {
+    UNSPECIFIEDx = 0;
+
+    STRICT = 1;
```

Comment:
Doc.  "Only if the set of endpoints in each target defined in routing_preference is empty or every endpoint is unreachable can an endpoint in the next target in routing_preference be used"